### PR TITLE
Fix ServiceNow wfs graphs and task descriptions

### PR DESCRIPTION
**Pull request by AlexFalessi**

### CODE REVIEW

**Review by aminelouati — APPROVED** [review]

File: ApplicationConnectors/resources/catalog/ServiceNow_Create_Task.xml [hunk 1]
```diff
@@ -163,27 +163,28 @@ println result
...
-REQUEST_BODY: a json parameter containing the request body. Example at https://developer.servicenow.com/dev.do#!/reference/api/quebec/rest/c_TableAPI#table-POST (Make sure to select the correct
-release name)
-DEBUG: to print the full request and response content in the task output
-RESPONSE_PATH: which data to extract in the response. It uses the GPath notation (https://groovy-lang.org/processing-xml.html)
-   </body>
+    <div id="workflow-visualization-view"><div id="workflow-visualization" style="position:relative;top:-318.6833190917969px;left:-499.10003662109375px"><div class="task _jsPlumb_endpoint_anchor_ ui-draggable" style="top: 323.683px; left: 504.1px;" id="jsPlumb_1_31"><a class="task-name" data-toggle="tooltip" data-placement="right" title="A task performing a rest POST request to create a task on a servicenow instance using a json request body.
+
+    It accepts the following parameters:
+      ENDPOINT: base url of the request (inherited from job variable)
+      USER: basic auth user for servicenow (inherited from job variable)
```
Comment (tao-xinxiu) [inline ApplicationConnectors/resources/catalog/ServiceNow_Create_Task.xml:176]: servicenow -> ServiceNow to be coherent.

File: ApplicationConnectors/resources/catalog/ServiceNow_Create_Task.xml [hunk 2]
```diff
@@ -163,27 +163,28 @@ println result
...
+      PASSWORD: basic auth password for servicenow (inherited from job variable)
```
Comment (tao-xinxiu) [inline ApplicationConnectors/resources/catalog/ServiceNow_Create_Task.xml:177]: servicenow -> ServiceNow

File: ApplicationConnectors/resources/catalog/ServiceNow_Mark_Task_As_Resolved.xml [hunk 3]
```diff
@@ -180,34 +180,33 @@ println result
...
-CONTENT_TYPE: content-type of the request, should be "application/json" with eventual encoding.
-REQUEST_BODY: a json parameter containing the request body. Example at https://developer.servicenow.com/dev.do#!/reference/api/quebec/rest/c_TableAPI#table-POST (Make sure to select the correct
-release name)
-DEBUG: to print the full request and response content in the task output
-RESPONSE_PATH: which data to extract in the response. It uses the GPath notation (https://groovy-lang.org/processing-xml.html)    </body>
+    <div id="workflow-visualization-view"><div id="workflow-visualization" style="position:relative;top:-318.6833190917969px;left:-499.10003662109375px"><div class="task _jsPlumb_endpoint_anchor_ ui-draggable" style="top: 323.683px; left: 504.1px;" id="jsPlumb_1_31"><a class="task-name" data-toggle="tooltip" data-placement="right" title="A task performing a rest PUT request to mark a task on a servicenow instance as resolved or closed.
+
+    It accepts the following parameters:
+      ENDPOINT: base url of the request (inherited from job variable)
+      USER: basic auth user for servicenow (inherited from job variable)
```
Comment (tao-xinxiu) [inline ApplicationConnectors/resources/catalog/ServiceNow_Mark_Task_As_Resolved.xml:193]: servicenow -> ServiceNow

File: ApplicationConnectors/resources/catalog/ServiceNow_Mark_Task_As_Resolved.xml [hunk 4]
```diff
@@ -180,34 +180,33 @@ println result
...
+      PASSWORD: basic auth password for servicenow (inherited from job variable)
```
Comment (tao-xinxiu) [inline ApplicationConnectors/resources/catalog/ServiceNow_Mark_Task_As_Resolved.xml:194]: servicenow -> ServiceNow

**Review by tao-xinxiu — APPROVED** [review]

**Pull request merged**